### ISSUE TITLE
Media player can't play .MOV video files

### ISSUE THREAD

**Issue by Stefano0101** [ubports/ubports-touch#249]
- Device:files Turbo
- Channel:Turbo 15.04/rc
- Build: 4

## Steps to reproduce

1. ...Open File manager and select a video with .MOV format
2. ...Media player trying to open the file
3. ...file can't be played due an error (format not supported)

## Expected behavoir

...

## Actual behavior

...

## Logfiles and additional information

...

**Comment by Stefano0101** [comment]
same on Nexus 5, 15.04, rc 5

**Comment by NeoTheThird** [comment]
There's already #7 reported for the Nexus 5, so let's keep these separate.

**Comment by dobey** [comment]
I think #7 is different from this. There videos play, but are rendered incorrectly. This is about a specific format not playing. @Stefano0101 can you please specify exactly what codecs are used inside the mov you are trying to play? Also, does this particular video play with stock AOSP install on the same devices?

**Comment by tomoqv** [comment]
Works for me on xenial.

**Comment by NeoTheThird** [comment]
@Stefano0101 Could you test this on xenial?

**Comment by Stefano0101** [comment]
the codecs from the .mov files trying to play: video H.264, audio DVI ADPCM,

**Comment by Stefano0101** [comment]
same on Xenial, revision 91, not working, file not supported, unfortunately. It's the footage from a dash cam, sadly they're using .mov file system. Other formats plays well.

**Comment by Flohack74** [comment]
Wasnt .mov also Apple Quicktime?

**Comment by Stefano0101** [comment]
Yes, I believe so.

**Comment by dobey** [comment]
@Stefano0101 Could you please attach a short video that doesn't show any personal info or such, for others to test with?

**Comment by Flohack74** [comment]
Well if your mov really contains Apple Quicktime format, then you need to check maybe if we even have this codec: https://help.ubuntu.com/community/RestrictedFormats - probably this is not in the standard distro packages.

**Comment by Stefano0101** [comment]
[FILE180611-123004.zip](https://github.com/ubports/ubuntu-touch/files/2094363/FILE180611-123004.zip)

**Comment by dobey** [comment]
@Flohack74 The `qtmux` gstreamer element is part of gstreamer-plugins-good, and so is on the phone. Which means it should be able to read the format, at least. However, gstreamer seems to have some issue decoding the audio/video streams for this video (even on 18.04 with gst-play-1.0 and more software decoders installed). There may be an issue with the decoding here and it's not getting the stream passed to hardware for decoding.

Ironically, it seems that a thumbnail does get created, so something at least was able to grab a frame somehow.

Anyway, I can confirm this bug is valid.

**Comment by Flohack74** [comment]
Does this affect all devices then?

**Comment by dobey** [comment]
Yes, I think so. I can't play the video in Totem on 18.04 on my PC either.
